### ISSUE TITLE
Provide option for watching files?

### ISSUE THREAD

**Issue by vysinsky** [jiripudil/intellij-nette-tester#12]
Do you plan to add support for `--watch` attribute? Or do you think it is better to use built-in file watchers and trigger tests from the file watcher?

**Comment by jkuchar** [comment]
What are you trying to achieve? How it should look like?

**Comment by fprochazka** [comment]
Imho he wants to run the tests automatically, on save.

**Comment by vysinsky** [comment]
Exactly. But I've realized that I can set file watcher. So it is probably not necessary to add it to the plug-in.

**Comment by jkuchar** [comment]
Please clould you provide tutorial/link how to do that? Thanks!

**Comment by jiripudil** [comment]
It would be nice to have it in the plugin, but Tester watches in one `while (TRUE) {}` process and implementation-wise, I have no clue how to tell that one run of the tests has ended or that another one has started, and re-create the test console for the next run.

**Comment by jkuchar** [comment]
I do not see any value in implementing `--watch` over FileWatchers in IDE. When you change code, tests run again. Result is the same. Do I miss something here?

**Comment by jiripudil** [comment]
File Watchers are meant for compiling various things like SCSS in the background, it even says so in [the docs](https://www.jetbrains.com/help/idea/2017.1/using-file-watchers.html#d464772e39). Thus, they can only launch binaries directly, not Run Configurations, so while yes, you can set a up a watcher to run Tester, all you get from it is a raw console output, without the added value of the plugin features. Result is the same as running `tester --watch` in the terminal.
